src/haz3lweb/view/CursorInspector.re

```diff
@@ -157,10 +157,19 @@ let typ_err_view = (ok: Info.error_typ) =>
     ]
   };
 
-let exp_view = (cls: Term.Cls.t, status: Info.status_exp) =>
+let rec exp_view = (cls: Term.Cls.t, status: Info.status_exp) =>
```

Comment:
`exp_view` is modified to be recursive since `error_exp` can be somehow "recursive" now